docker-compose-configuration/rds-local-apps/rds-local/config/shiro.ini

```diff
@@ -0,0 +1,5 @@
+[users]
```

Comment:
please remove shiro.ini, this should use the default platform credentials (admin/admin)
Please also add a hint to the readme regarding logging in as asmin/admin and then changing the password on `Admin:Security`

Reply:
done.